src/main/java/com/logistic/project/configuration/MongoConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.logistic.project.configuration;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+@Configuration
+public class MongoConfig {
```

Comment:
If you use spring boot, which you do, you don't need this custom config. Spring boot automatically declares mpngotemplate, provided that there are necessary parameters in application.properties